CHANGELOG.md

```diff
@@ -10,10 +10,17 @@ and this project adheres to
 
 ### Added
 
+- [#4875](https://github.com/firecracker-microvm/firecracker/pull/4875):
+  Increase default queue size for the `virtio-net` device from 256 to 512. This
+  decreases wait time between guest and vmm threads for network packets
+  processing and allows for more throughput.
 - [#4834](https://github.com/firecracker-microvm/firecracker/pull/4834): Add
   `VIRTIO_NET_F_RX_MRGBUF` support to the `virtio-net` device. When this feature
   is negotiated, guest `virtio-net` driver can perform more efficient memory
   management which in turn improves RX and TX performance.
+- [#4844](https://github.com/firecracker-microvm/firecracker/pull/4844): Upgrade
+  `virtio-net` device to use `readv` syscall to avoid unnecessary memory copies
+  on RX path, increasing the RX performance.
```

Comment:
Is "RX" something that everyone will understand here? Or should we put "host to guest throughput" instead, or something?

Reply:
Also, these should arguably both be under "Changed", not "Added"

Reply:
We already have `RX` and `TX` in the there. And these are standard names as far as I know.

Reply:
moved to `Changed`